src/arrow-sdk.ts

```diff
@@ -190,17 +190,74 @@ export async function getRecommendedStrategies(
                 price_history: priceHistory
             }
         )
-        const recommendedOptions = recommendedOptionResponse.data.options.map((option: any) => {
-            return {
+
+        const firstRecommendedOption = recommendedOptionResponse.data.options.max_profit
```

Comment:
Maybe this doesn't make sense but is it possible to just return an array of options from the API that we can parse in some specified order? Might be hard/weird so I would understand if we can't do that.

Reply:
Hmmm yea, like an array that always has the first element as the max profit, second element as the first min losses, and the third element as the second min losses. I can make that update on the API and make the SDK updates

Reply:
@tenax002 Unless we always think the recommender will stay somewhat similar, I think that makes sense. Go ahead and make a Linear task as well. You can put that in backlog for now.